packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContext/DataTreeContext.js

```diff
@@ -0,0 +1,186 @@
+import React, { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+//sentene variant
+export const sampleDataStructure_sentence = {
```

Comment:
anything that is considered sample or for the purposes of the storybook demo should be moved into a separate file to ensure a separation of user and product code. a folder called `assets` or something similar would be a great place to put these.

Reply:
Moved to assets folder. Sample input is the inputConfig expected as input prop. SampleData is the samples of data tree structure we are building . All updates like add condition , modify condition , remove condition will update the rootState object and renders the component  accordingly